# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/mozilla/home-bookmarks.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xpfe/browser/resources/locale/en-US/region.properties.orig	Sat May 25 05:01:32 2002
+++ ./xpfe/browser/resources/locale/en-US/region.properties	Sat May 25 05:04:23 2002
@@ -1,5 +1,5 @@
 # navigator.properties
-homePageDefault=http://www.mozilla.org/
+homePageDefault=http://www.t2-project.org/startpage/
 shopKeyword=keyword:shop [Product]
 quoteKeyword=keyword:quote [Enter symbol here]
 localKeyword=keyword:zip [Your zip code]
@@ -11,7 +11,7 @@
 #
 # all.js
 #
-browser.startup.homepage=http://www.mozilla.org/start/
+browser.startup.homepage=http://www.t2-project.org/startpage/
 browser.throbber.url=http://www.mozilla.org/
 browser.search.defaulturl=http://search.netscape.com/cgi-bin/search?search=
 
@@ -20,4 +20,4 @@
 
 #config.js
 #
-startup.homepage_override_url=http://www.mozilla.org/start/
+startup.homepage_override_url=http://www.t2-project.org/startpage/
